provisioner/cmd/main.go

```diff
@@ -45,6 +45,7 @@ var (
 
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
+	klog.InitFlags(nil)
```

Comment:
I remember there was some strange ordering with klog and flags that we had to deal with with csi sidecars.  See an [example](https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go#L76)

Reply:
The ordering is simple:

- register CLI flags on flagset (we use go default flagset here)
- parse flags
- override flags (e.g. enable `logtostderr` always)

Your example is complex because it uses `pflag` to parse CLI flags, so it must register go standard flagset onto `pflag` flagset and call `Parse` on `pflag`.